src/Explorer/Controls/NotebookGallery/InfoComponent/InfoComponent.less

```diff
@@ -0,0 +1,26 @@
+@import "../../../../../less/Common/Constants.less";
+.infoPanel, .infoPanelMain {
+    display: flex;
+    align-items: center;
+}
+
+.infoPanel {
+    padding-left: 5px;
+    padding-right: 5px;
+}
+
+.infoLabel, .infoLabelMain {
+    padding-left: 5px  
+}
+
+.infoLabel {
+    font-weight: 400
```

Comment:
reuse constants as much as possible

Reply:
I can't find any constants for font weight as of now.